orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -251,20 +256,20 @@ def forward(self, *inputs, **kwargs):
 
             self._gradient_accumulation_manager.maybe_update_cache_before_run()
 
+            prepared_input_list, _, _ = _io._combine_input_buffers_initializers(
+                self._graph_initializers,
+                self._graph_info.user_input_names,
+                self._input_info,
+                self._flattened_module.named_buffers(),
+                inputs,
+                kwargs,
+                self._device,
+                self._rt_inspector.input_density_ob if self._print_input_density else None,
```

Comment:
Maybe I missed something here; if `self._print_input_density` is not `True`, then we compute the `embed_sparsity_results` and `label_sparsity_results` on each loop even though we don't use it?